src/xrPhysics/DisablingParams.h

```diff
@@ -28,6 +27,4 @@ struct SAllDDWParams
     float reanable_factor;
 };
 
-extern SAllDDWParams worldDisablingParams;
-
-#endif
+extern XRPHYSICS_API SAllDDWParams worldDisablingParams;
```

Comment:
Since DisablingParams is only used in xrPhysics, `XRPHYSICS_API` is not needed.
```suggestion
extern SAllDDWParams worldDisablingParams;
```